modules/motions/validation/validateSDVTNodeOperatorNamesSet.ts

```diff
@@ -0,0 +1,22 @@
+import { SetNodeOperatorNamesAbi } from 'generated'
+import { MotionTypeForms } from '../types'
+import { getConcurrentMotionsErrorText } from './getConcurrentMotionsErrorText'
+import { MotionValidator } from './types'
+import { validateConcurrentMotions } from './validateConcurrentMotions'
+
+export const validateSDVTNodeOperatorNamesSet: MotionValidator = async args => {
+  // Check MF0406: No same names assignment in concurrent motions
+  const activeMotionsValidation =
+    await validateConcurrentMotions<SetNodeOperatorNamesAbi>({
+      motionType: MotionTypeForms.SDVTNodeOperatorNamesSet,
+      callDataKey: 'name',
+      formDataKey: 'name',
+      ...args,
```

Comment:
I think that better use `...args` on to object creation, to be sure that no key reassign (if reassign not planning)